Blockzilla/AboutViewController.swift

```diff
@@ -159,15 +174,41 @@ private class AboutHeaderView: UIView {
         aboutParagraph.textColor = UIConstants.colors.defaultFont
         aboutParagraph.font = UIConstants.fonts.aboutText
         aboutParagraph.numberOfLines = 0
-        addSubview(aboutParagraph)
+        return aboutParagraph
+    }()
 
+    private lazy var learnMoreButton : UIButton = {
         let learnMoreButton = UIButton()
         learnMoreButton.setTitle(UIConstants.strings.aboutLearnMoreButton, for: .normal)
         learnMoreButton.setTitleColor(UIConstants.colors.settingsLink, for: .normal)
         learnMoreButton.setTitleColor(UIConstants.colors.buttonHighlight, for: .highlighted)
         learnMoreButton.titleLabel?.font = UIConstants.fonts.aboutText
         learnMoreButton.addTarget(self, action: #selector(didPressLearnMore), for: .touchUpInside)
+        return learnMoreButton
+    }()
+
+    init() {
```

Comment:
If we change this to 
```
convenience init() {
    self.init(frame: .zero)
    addSubviews()
    configureConstraints()
```

We can get rid of the `required init?` underneath :)

Reply:
Of course. Fixed.